tests/e2e/specs/log.cy.js

```diff
@@ -16,92 +16,70 @@
  */
 
 describe('Log View', () => {
-  it('works', () => {
-    cy
-      .visit('/#/log/one')
+  beforeEach(() => {
+    cy.visit('/#/log/one')
+  })
 
-      // the workflow ID should be filled in
-      .get('.workflow-id-input')
-      .find('input')
+  it('fills in inputs', () => {
+    // the workflow ID should be filled in
+    cy.get('.c-log [data-cy=workflow-id-input]')
       .should('be.visible')
-      .should('have.attr', 'disabled', 'disabled')
-      .should(($input) => {
-        expect($input.val()).to.equal('~user/one')
-      })
-
-      // the job log files list should have been populated by the query
-      .get('.c-log')
-      .should(($el) => {
-        expect($el[0].__vue__.logFiles).to.have.length(3)
```

Comment:
This doesn't work in Vue 3, and is better suited to unit testing that e2e Cypress testing